src/tests/running.md

```diff
@@ -242,7 +242,7 @@ machine, e.g. for RISC-V
 ```
 
 The binary will be created at
-`./build/$HOST_ARCH/stage2-tools/$TARGET_ARCH/release/remote-test-server`. Copy
+`./build/$HOST_ARCH/stage2-tools/host/release/remote-test-server`. Copy
```

Comment:
Hmm, is this change correct? Even if it is, I think it might be clearer to leave it explicit.

Reply:
oops, I typo-d this, ty.

I think it's better to use `host/` here because it makes it more clear what bootstrap considers the host and what it considers the target; you can always use `ls -l build/host` to see where it points.

Reply:
Ok, sounds good to me.